AchieveIt.BusinessLogic/Profiles/UserProfile.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AchieveIt.BusinessLogic.DTOs.Auth;
+using AchieveIt.DataAccess.Entities;
+using AutoMapper;
+
+namespace AchieveIt.BusinessLogic.Profiles
+{
+    public class UserProfile : Profile
+    {
+        public UserProfile()
```

Comment:
Moreover, if we will have inheritance (like we have in the Entities), we will be able to use mapping inheritance as well.
![image](https://user-images.githubusercontent.com/45832778/149520374-c76562db-4c97-47f0-ab9c-9ec79c94c823.png)

Reply:
Done.